content/version/2/0/code_of_conduct.nl.md

```diff
@@ -0,0 +1,93 @@
+* "Trollen", beledigende of denigrerende opmerkingen, en persoonlijke of politieke aanvallen
+* Openbare of privé-intimidatie
+* Publiceren van persoonlijke informatie van anderen, zoals een fysiek adres of e-mail adres, zonder hun uitdrukkelijke toestemming
+* Ander gedrag dat redelijkerwijs als ongepast zou kunnen worden beschouwd in een professionele setting
+
+## Handhaving
+
+Gemeenschapsleiders zijn verantwoordelijk voor het verduidelijken en handhaven van onze normen van acceptabel gedrag en zal passende en eerlijke corrigerende maatregelen nemen in reactie op elk gedrag dat zij ongepast, bedreigend, aanstootgevend, of schadelijk vinden.
+
+Gemeenschapsleiders hebben het recht en de verantwoordelijkheid om opmerkingen, commits, code, wiki-bewerkingen, issues en andere bijdragen die niet in overeenstemming met deze Gedragscode zijn te verwijderen, bewerken of afwijzen, en zal redenen voor moderatie meedelen waar nodig.
+
+## Toepassingsgebied
+
+Deze gedragscode is van toepassing binnen alle gemeenschappelijke ruimtes, en is ook van toepassing wanneer een individu officieel de gemeenschap in openbare ruimtes vertegenwoordigt.
+Voorbeelden van het vertegenwoordigen van onze gemeenschap zijn onder meer het gebruik van een officieel e-mailadres, posten via een officieel social media-account, of optreden als een aangestelde vertegenwoordiger bij een online of offline evenement.
+
+## Handhaving
+
+Beledigend, intimiderend of anderszins onaanvaardbaar gedrag kan gerapporteerd worden aan de gemeenschapsleiders die verantwoordelijk zijn voor handhaving via [CONTACTMETHODE INVOEGEN].
+Alle klachten zullen onmiddellijk en eerlijk worden beoordeeld en onderzocht.
+
+Alle gemeenschapsleiders zijn verplicht om de privacy en veiligheid van de verslaggever van een incident te respecteren.
+
+## Handhavingsrichtlijnen
+
+Gemeenschapsleiders zullen onderstaande "Community Impact Guidelines" volgen bij het bepalen van de consequenties van elke handeling die zij in strijd met de Gedragscode achten:
+
+### 1. Correctie
+
+**Impact op de gemeenschap**: gebruik van ongepast taalgebruik of ander gedrag dat onprofessioneel wordt geacht, of niet welkom in de gemeenschap.
+
+**Gevolg**: een persoonlijke, schriftelijke waarschuwing van community leiders, verduidelijking over de aard van de overtreding en een uitleg waarom het gedrag ongepast wordt geacht. Mogelijk wordt om een ​​openbare verontschuldiging gevraagd.
+
+### 2. Waarschuwing
+
+**Impact op de gemeenschap**: een overtreding door een enkel incident of reeks van acties.
+
+**Consequentie**: een waarschuwing met gevolgen als het gedrag voortduurt. Geen interactie met de betrokken personen, inclusief ongevraagde interactie met degenen die de gedragscode handhaven, voor een bepaalde periode. Dit omvat het vermijden van interacties in gemeenschappelijke ruimtes en externe kanalen zoals op social media. Het overtreden van deze voorwaarden kan leiden tot een tijdelijke of permanente ban.
```

Comment:
~Consequentie~ **Gevolg**

Reply:
Consequentie is a beautiful word I think

Reply:
Yes, it is a beautiful word! But all the other sections describe **Gevolg** 😅. Or change everything to **Consequentie**.

Reply:
As an international foreigner I would be biased to "change everything to **Consequentie**".